src/views/Admin/ReportingTool.vue

```diff
@@ -546,6 +573,90 @@ export default {
         },
       ];
     },
+    getOrderBarChartData() {
+      this.OrderBarChart.data = {
+        labels: this.getOrderDates(),
+        datasets: [
+          {
+            barPercentage: 1,
+            barThickness: 12,
+            maxBarThickness: 16,
+            label: "Number of sales made per day",
+            data: this.getOrderValues(),
+            backgroundColor: [
+              "#6ae299",
+              "#5973e5",
+              "#ebad4c",
+              "#fb8ca0",
+              "#2582c8",
+              "#13098c",
+              "#1e023b",
+              "#510387",
+              "#20bc00",
+              "#f5485f",
+              "#73b6ad",
+              "#dbc620",
+              "#046b2d",
+              "#423f5a",
+              "#e80130",
+              "#cb997e",
+              "#ff0a54",
+              "#b392ac",
+              "#355070",
+              "#be0aff",
+              "#4d1230"
+            ],
+            borderColor: [
+              "#6ae299",
+              "#5973e5",
+              "#ebad4c",
+              "#fb8ca0",
+              "#2582c8",
+              "#13098c",
+              "#1e023b",
+              "#510387",
+              "#20bc00",
+              "#f5485f",
+              "#73b6ad",
+              "#dbc620",
+              "#046b2d",
+              "#423f5a",
+              "#e80130",
+              "#cb997e",
+              "#ff0a54",
+              "#b392ac",
+              "#355070",
+              "#be0aff",
+              "#4d1230"
+            ],
+            borderWidth: 1,
+          },
+        ],
+      };
+    },
+    getOrderDates() {
+      const dateConfig = {
+        weekday: "short",
+        year: "2-digit",
+        month: "short",
+        day: "numeric",
+      };
+      return this.reports.meta.totalSalesPerDay.map((item) => {
+        let date = new Date(item.order_date);
+        let month = date.toLocaleString("en-us", { month: dateConfig.month });
+        let weekday = date.toLocaleString("en-us", {
+          weekday: dateConfig.weekday,
+        });
+        let year = date.toLocaleString("en-us", { year: dateConfig.year });
+        let day = date.toLocaleString("en-us", { day: dateConfig.day });
+        return [weekday, month, year].join("-");
+      }) ?? null;
+    },
+    getOrderValues() {
+      return this.reports.meta.totalSalesPerDay.map((item) => {
```

Comment:
rewrite line 656 `return this.reports?.meta?.totalSalesPerDay?.map(item =>  item.total);`